source/apiVolontaria/apiVolontaria/settings.py

```diff
@@ -26,7 +26,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['localhost']
```

Comment:
This change is not necessary and can be a limitation to special configuration, please remove it.

```
When DEBUG is True and ALLOWED_HOSTS is empty, the host is validated against ['localhost', '127.0.0.1', '[::1]'].
```

Source documentation : https://docs.djangoproject.com/en/2.1/ref/settings/#allowed-hosts